### ISSUE TITLE
Support for Cloudflare Workers

### ISSUE THREAD

**Issue by dijitali** [terraform-providers/terraform-provider-cloudflare#50]
Hi there,

It would be awesome to have support for the recently released Cloudflare Workers from Terraform (https://blog.cloudflare.com/cloudflare-workers-unleashed/).

### Terraform Version
```
$ terraform -v
Terraform v0.11.7
```

### Affected Resource(s)
Potential resources could be of the form:
```
resource "cloudflare_worker" "foobar" {
  zone_id = "${var.cloudflare_zone_id}"
  filter = "example.com/*"
  script_content = ${file("path.txt")}
}
```

### Expected Behavior
Terraform provisions a Cloudflare Worker resource.

### Actual Behavior
Not supported yet.

### References
https://developers.cloudflare.com/workers/api/

**Comment by prdonahue** [comment]
It's on our roadmap, but don't have a timeline to share right now.

**Comment by jRiest** [comment]
This is being worked on in #111

**Comment by jRiest** [comment]
This is released in 1.4.0
